### PR TITLE
Added missing Uzbek translations.

### LINKED ISSUE #51962: Missing translations for Uzbek (uz)
Hello,

There are some missing translation for Uzbek (uz). These should be added in branch 5.4. 
See https://github.com/symfony/symfony/issues/38710 for details and [this page](https://symfony-translations.nyholm.tech/#pr) for an example.

These are the files that should be updated: 
- [src/Symfony/Component/Validator/Resources/translations/validators.uz.xlf](https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Validator/Resources/translations/validators.uz.xlf)

> [!NOTE]
> If you want to work on this issue, add a comment to assign it to yourself and let others know that this is already taken.

### PR DESCRIPTION
**Pull request by asrorbekh**
| Q             | A
| ------------- | ---
| Branch?       | 5.4
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Tickets       | Fix #51962
| License       | MIT

Added missing translations (translation units 104-109) and fixed some translations in previous units.

### CODE REVIEW

File: src/Symfony/Component/Validator/Resources/translations/validators.uz.xlf [hunk 1]
```diff
@@ -4,11 +4,11 @@
         <body>
             <trans-unit id="1">
                 <source>This value should be false.</source>
-                <target>Qiymat noto'g'ri bo'lishi kerak.</target>
+                <target>Qiymat false bo'lishi kerak.</target>
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Validator/Resources/translations/validators.uz.xlf:7]: why this change? `yaroqsiz`?

File: src/Symfony/Component/Validator/Resources/translations/validators.uz.xlf [hunk 2]
```diff
@@ -4,11 +4,11 @@
...
             </trans-unit>
             <trans-unit id="2">
                 <source>This value should be true.</source>
-                <target>Qiymat to'g'ri bo'lishi kerak.</target>
+                <target>Qiymat true bo'lishi kerak.</target>
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Validator/Resources/translations/validators.uz.xlf:11]: yaroqli?

File: src/Symfony/Component/Validator/Resources/translations/validators.uz.xlf (hunk 1 above)
Comment (asrorbekh) [reply src/Symfony/Component/Validator/Resources/translations/validators.uz.xlf:7]: Technical words should not be translated if i am not mistaken.

**Comment by nicolas-grekas** [comment]
Thank you @asrorbekh.
